android/src/main/java/com/clevertap/clevertap_plugin/ClevertapCustomTemplates.java

```diff
@@ -0,0 +1,75 @@
+package com.clevertap.clevertap_plugin;
```

Comment:
Since Kotlin is now included in the project, I think it would be best to have all new classes in Kotlin. The RN implementation can used for references: https://github.com/CleverTap/clevertap-react-native/blob/master/android/src/main/java/com/clevertap/react/CleverTapCustomTemplates.kt

Reply:
done.